tests/Microsoft.Identity.Test.Unit/AppConfigTests/PublicClientApplicationBuilderTests.cs

```diff
@@ -649,7 +649,7 @@ public void IsBrokerAvailable_OldDotNet()
 
```

Comment:
Do we have full test coverage to test that correct broker implementation is used for all platforms? Do we test net6_win anywhere? We can add more tests here (or wherever is more appropriate).